runtime/drivers/sqlite/migrations/0008.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE instances ADD COLUMN olap_connector TEXT NOT NULL DEFAULT '';
+UPDATE instances SET olap_connector = olap_driver;
+ALTER TABLE instances DROP COLUMN olap_driver; 
+
+ALTER TABLE instances ADD COLUMN repo_connector TEXT NOT NULL DEFAULT '';
+UPDATE instances SET repo_connector = repo_driver;
+ALTER TABLE instances DROP COLUMN repo_driver;
```

Comment:
Why not just `RENAME COLUMN`?

Reply:
OOPS. I forgot `RENAME COLUMN` is also available.